.github/workflows/CreateGitHubPagesForR.yml

```diff
@@ -0,0 +1,81 @@
+name: Create GitHub pages for an R package
+
+on: 
+  workflow_call:
+    inputs:
+      pkgdown_mode:
+        required: true
+        type: string
```

Comment:
must be one of 'devel' or 'release'
Unfortunately it seems to be no possibility to directly check the input enum values (the [choice type](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype) is not available for this type of workflow - not sure why)